daml-lf/spec/daml-lf-1.rst

```diff
@@ -3667,35 +3670,51 @@ Numeric functions
 
 * ``ADD_NUMERIC : ∀ (α : nat) . 'Numeric' α → 'Numeric' α  → 'Numeric' α``
 
-  Adds the two decimals.  The scale of the inputs and the output is
-  given by the type parameter `α`.  Throws an error in case of
-  overflow.
+   Shortcut for::
+
+      'ADD_NUMERIC' ≡
+        Λ α : ⋆. λ x : 'Numeric' α . λ : 'Numeric' α .
+	        let x' = 'TO_BIGDECIMAL_NUMERIC' α 'UnBondedMathContext' x in
+            let y' = 'TO_BIGDECIMAL_NUMERIC' α 'UnBondedMathContext' y in
+            let r  = ``ADD_BIGDECIMAL`` 'UnBondedMathContext' x' y' in
+            'TO_NUMERIC_BIGDECIMAL' α RoundingHalfEven r
+
 
 * ``SUB_NUMERIC : ∀ (α : nat) . 'Numeric' α → 'Numeric' α → 'Numeric' α``
 
-  Subtracts the second decimal from the first one.  The
-  scale of the inputs and the output is given by the type parameter
-  `α`.  Throws an error if overflow.
+  Shortcut for::
+
+      'SUB_NUMERIC' ≡
+        Λ α : ⋆. λ x : 'Numeric' α . λ : 'Numeric' α .
+	        let x' = 'TO_BIGDECIMAL_NUMERIC' α 'UnBondedMathContext' x in
+            let y' = 'TO_BIGDECIMAL_NUMERIC' α 'UnBondedMathContext' y in
+            let r  = ``SUB_BIGDECIMAL`` 'UnBondedMathContext' x' y' in
+            'TO_NUMERIC_BIGDECIMAL' α RoundingHalfEven r
 
 * ``MUL_NUMERIC : ∀ (α₁ α₂ α : nat) . 'Numeric' α₁ → 'Numeric' α₂ → 'Numeric' α``
 
-  Multiplies the two numerics and rounds the result to the closest
-  multiple of ``10⁻ᵅ`` using `banker's rounding convention
-  <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>`_.
-  The type parameters `α₁`, `α₂`, `α` define the scale of the first
-  input, the second input, and the output, respectively. Throws an
-  error in case of overflow.
+ Shortcut for::
+
+      'MUL_NUMERIC' ≡
+        Λ α₁ : ⋆.  Λ α₂ : ⋆. Λ α : ⋆. λ x : 'Numeric' α₁ . λ : 'Numeric' α₂ .
+	        let x' = 'TO_BIGDECIMAL_NUMERIC' α₁ 'UnBondedMathContext' x in
+            let y' = 'TO_BIGDECIMAL_NUMERIC' α₂ 'UnBondedMathContext' y in
+            let r  = ``MUL_BIGDECIMAL`` 'UnBondedMathContext' x' y' in
+            'TO_NUMERIC_BIGDECIMAL' α RoundingHalfEven r
 
 * ``DIV_NUMERIC : ∀ (α₁ α₂ α : nat) . 'Numeric' α₁ → 'Numeric' α₂ → 'Numeric' α``
 
-  Divides the first decimal by the second one and rounds the result to
-  the closest multiple of ``10⁻ᵅ`` using `banker's rounding convention
-  <https://en.wikipedia.org/wiki/Rounding#Round_half_to_even>`_ (where
-  `n` is given as the type parameter).  The type parameters `α₁`,
-  `α₂`, `α` define the scale of the first input, the second input, and
-  the output, respectively. Throws an error in case of overflow.
+      'DIV_NUMERIC' ≡
+        Λ α₁ : ⋆.  Λ α₂ : ⋆. Λ α : ⋆. λ x : 'Numeric' α₁ . λ : 'Numeric' α₂ .
+	        let x' = 'TO_BIGDECIMAL_NUMERIC' α₁ 'UnBondedMathContext' x in
+            let y' = 'TO_BIGDECIMAL_NUMERIC' α₂ 'UnBondedMathContext' y in
+            let r  = ``DIV_BIGDECIMAL`` 'UnBondedMathContext' x' y' in
```

Comment:
this is not correct. Unbounded throws if the result cannot be represented exactly. You need to round here to match semantics of Numeric.